### ISSUE TITLE
How to reproduce rFID

### ISSUE THREAD

**Issue by sen-ye** [bytetriper/RAE#11]
Thank you for this great work!
I'm having difficulty reproducing the reported rFID results. Here are my results:

Setup:
- Model: DINOv2-B-ViT-XL
- Evaluation framework: OpenAI guided diffusion
- Reference: `VIRTUAL_imagenet256_labeled.npz`

Results:
- IS: 224.64
- FID: 1.90
- sFID: 5.91

Could you please clarify:
1. The correct reference dataset for rFID computation?
2. Any specific evaluation settings or preprocessing steps?
3. The evaluation code used?

**Comment by Y-Sisyphus** [comment]
> Thank you, I have got the correct results: PSNR: 18.9438, rFID: 0.5776

Hello, in Table 1(a) of the paper, the rFID of DINOv2-B-ViT-XL is reported as 0.49. I am curious about the reason for this discrepancy compared to your results.

**Comment by sen-ye** [comment]
> > Thank you, I have got the correct results: PSNR: 18.9438, rFID: 0.5776
> 
> Hello, in Table 1(a) of the paper, the rFID of DINOv2-B-ViT-XL is reported as 0.49. I am curious about the reason for this discrepancy compared to your results.

You can check Table 5 in the paper.

**Comment by shreyas1230** [comment]
> VIRTUAL_imagenet256_labeled.npz is the reference batch for computing gFID, not for rFID.

~~What is the reference batch for computing rFID? I am trying to replicate rFID results with DINOv2 and ViTXL, but am consistently getting FID values of 1.83 on the validation set.~~

I ended up getting FID=0.57025 by taking the ImageNet-1k validation set, center cropping it to be 256x256 as advised by https://github.com/bytetriper/RAE/issues/11#issuecomment-3404963731 (had to write a custom script for this as I didn't find one in the repo), and saved it to a flattened directory. Then I was able to compute FID using `pytorch-fid`.

A few questions:
- Why doesn't VIRTUAL_imagenet256_labeled.npz work as a reference for rFID? Ideally, the distribution of the images (whether reconstructed with a decoder or generated using a DiT) should match the original ImageNet distribution. Or am I missing something here?
- Though I achieved 0.57 rFID, I wasn't able to reproduce the 0.5776 value. Did you do something else to get the rFID value?
- Are the pretrained checkpoints available for ViT decoders trained without additive noise?
